bin/crew

```diff
@@ -580,16 +580,24 @@ end
 
 def updatefiles
   puts 'Rebuilding crew files cache...'.yellow
-  FileUtils.rm_rf "#{CREW_MANIFEST_CACHE_DIR}/*"
-  # if CREW_LOCAL_MANIFEST_PATH.empty?
-  Dir.chdir CREW_MANIFEST_CACHE_DIR do
-    if ENV['CREW_TESTING'] == '1'
-      system "curl -L#O https://github.com/#{CREW_TESTING_ACCOUNT}/chromebrew/archive/#{CREW_TESTING_BRANCH}.tar.gz"
-      system "tar x#{@verbose}f #{CREW_TESTING_BRANCH}.tar.gz chromebrew-#{CREW_TESTING_BRANCH}/manifest"
-    else
-      system "curl -L#O https://github.com/#{CREW_GITHUB_ACCOUNT}/chromebrew/archive/#{CREW_GITHUB_BRANCH}.tar.gz"
-      system "tar x#{@verbose}f #{CREW_GITHUB_BRANCH}.tar.gz chromebrew-#{CREW_GITHUB_BRANCH}/manifest"
+  if File.writable?(CREW_MANIFEST_CACHE_DIR)
+    # Be careful when removing items in a CREW_MANIFEST_CACHE_DIR crew
+    # has been pointed to.
+    FileUtils.rm_rf Dir.glob("#{CREW_MANIFEST_CACHE_DIR}/*.tar.gz")
+    FileUtils.rm_rf Dir.glob("#{CREW_MANIFEST_CACHE_DIR}/*/manifest")
```

Comment:
Why do we need to be careful with removal?  Does `FileUtils.rm_rf Dir["#{CREW_MANIFEST_CACHE_DIR}/*"]` work?

Reply:
Because `CREW_MANIFEST_CACHE_DIR=/usr/local crew updatefiles` will delete everything under that dir.

Reply:
Unless we want that to be a hidden option? I could be convinced...😅

Reply:
Would it make sense to only be careful if `CREW_MANIFEST_CACHE_DIR` is set by an environment variable?  Either way, not doing a clean sweep is prone to leave residue behind.